blocks/init/src/blocks/components/hamburger/hamburger-style.scss

```diff
@@ -0,0 +1,78 @@
+/* stylelint-disable max-nesting-depth */
+
+$hamburger: (
+  colors: (
+    line: #2C2C2C,
+  ),
+  width: 24px,
+  line-height: 2px,
```

Comment:
the same as up

Reply:
See https://github.com/infinum/eightshift-frontend-libs/pull/83#discussion_r382570466